assets/xml/objects/object_fish.xml

```diff
@@ -130,7 +130,7 @@
         <DList Name="gFishingOwnerLeftHandDL" Offset="0x83B8" />
 
         <!-- Owner Limbs and Skel -->
-        <Limb Name="gFishingOwnerTorsoLimb" Type="Standard" EnumName="FISHING_OWNER_LIMB_TORSO" Offset="0x8578" />
+        <Limb Name="gFishingOwnerTorsoLimb" Type="Standard" EnumName="FISHING_OWNER_LIMB_MIDDLE_SEGMENT" Offset="0x8578" />
```

Comment:
Looks like this was changed on accident? Probably an accidental find-and-replace